code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -13,6 +13,10 @@
 	openicon = "fryer_open"
 	has_specials = TRUE
 	upgradeable = TRUE
+	/// How many times have we deep fried ourself?
+	var/deepfried = 0
+	/// Total times it can be cmagged
+	var/cmag_limit = 150
```

Comment:
can this just be turned into a define, I see no reason for this to be a variable or be edited